src/acceptance/assets/app/go_app/README.md

```diff
@@ -0,0 +1,63 @@
+# Deploy Test Application with Custom Metrics
```

Comment:
```suggestion
# ASATS test app

This app is used in the app-autoscaler acceptance test suite (ASATS) to generate metrics to test end-to-end scenarios.

The app can be made to consume and release on demand:
- main memory 
- CPU
- hard disk

It can be used to emit custom metrics.

It can be used to respond to HTTP requests:
- as fast as possible
- or with a configurable request latency

**The [API](https://github.com/cloudfoundry/app-autoscaler-release/blob/main/src/acceptance/assets/app/go_app/internal/app/app.go#L49-L54) is not part of the public API and may change at any time.**

Still, the app can be useful during development and one such use case is described in detail in the next section.

# Deploy Test Application with Custom Metrics
```